src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1PlacementApplicationDomainEventService.kt

```diff
@@ -42,7 +42,13 @@ class Cas1PlacementApplicationDomainEventService(
       deliusEventNumber = application.eventNumber,
       withdrawnAt = eventOccurredAt,
       withdrawnBy = domainEventTransformer.toWithdrawnBy(user),
-      withdrawalReason = placementApplication.withdrawalReason!!.name
+      withdrawalReason = placementApplication.withdrawalReason!!.name,
+      placementDates = placementApplication.placementDates.map {
```

Comment:
Are we going to need to backfill these? Or are we not really concerned?

Reply:
These domain events have only been raised in pre-prod so happy to not bother in this case